src/process_hru_shapes.R

```diff
@@ -3,11 +3,11 @@ library(lwgeom)
 #proj_string <- '+proj=aea +lat_1=29.5 +lat_2=45.5 +lat_0=37.5 +lon_0=-96 +x_0=0 +y_0=0 +ellps=GRS80 +datum=NAD83 +units=m'
 proj_string <- 4326
 
-gfdb <- "cache/GF_nat_reg.gdb"
-hru_reduced <- read_sf(gfdb, "nhru")  %>% 
-  dplyr::select(Shape, hru_id_nat) %>% 
+gfdb <- "cache/GFv1.1.gdb"
+hru_reduced <- read_sf(gfdb, "nhru_v1_1_simp")  %>% 
```

Comment:
the new non simplified version of the GF is too large to export as a shp and the output ends up missing features, so I opted to use their simplified version which works great.